api/v1alpha1/edgedevicesignedrequests_types.go

```diff
@@ -36,6 +36,9 @@ type EdgeDeviceSignedRequestSpec struct {
 	// TargetNamespace is the namespace where device will land
 	TargetNamespace string `json:"target_namespace"`
 
+	// TargetSet is set that edgedevice will use.
+	TargetSet string `json:"target_set,omitempty"`
```

Comment:
I haven't paid enough attention before - the convention of json properties for CRDs is camelCase; please adjust `EdgeDeviceSignedRequest` (that includes `target_namespace`).